flightsql/src/main/java/io/deephaven/server/flightsql/FlightSqlResolver.java

```diff
@@ -0,0 +1,1916 @@
+        @Override
+        void checkForGetInfo(CommandGetImportedKeys command) {
+            if (CommandGetImportedKeys.getDefaultInstance().equals(command)) {
+                // TODO: Plumb through io.deephaven.server.arrow.FlightServiceGrpcImpl.getSchema
+                // We need to pretend that CommandGetImportedKeys.getDefaultInstance() is a valid command until we can
+                // plumb getSchema through to the resolvers.
+                return;
+            }
+            if (!hasTable(
+                    command.hasCatalog() ? command.getCatalog() : null,
+                    command.hasDbSchema() ? command.getDbSchema() : null,
+                    command.getTable())) {
+                throw tableNotFound();
+            }
+        }
+
+        // No need to check at resolve time since there is no actual state involved. If Deephaven exposes the notion
+        // of keys, this will need to behave more like QueryBase where there is a handle-based ticket and some sort
+        // state maintained. It is also incorrect to perform the same checkForFlightInfo at resolve time because the
+        // state of the server may have changed between getInfo and doGet/doExchange, and getInfo should still be valid
+        // for client.
+        // @Override
+        // void checkForResolve(CommandGetImportedKeys command) {
+        // }
+    };
+
+    private final CommandHandler commandGetExportedKeysHandler = new CommandStaticTable<>(CommandGetExportedKeys.class,
+            CommandGetKeysConstants.TABLE, FlightSqlTicketHelper::ticketFor) {
+        @Override
+        void checkForGetInfo(CommandGetExportedKeys command) {
+            if (CommandGetExportedKeys.getDefaultInstance().equals(command)) {
+                // TODO: Plumb through io.deephaven.server.arrow.FlightServiceGrpcImpl.getSchema
+                // We need to pretend that CommandGetExportedKeys.getDefaultInstance() is a valid command until we can
+                // plumb getSchema through to the resolvers.
+                return;
+            }
+            if (!hasTable(
+                    command.hasCatalog() ? command.getCatalog() : null,
+                    command.hasDbSchema() ? command.getDbSchema() : null,
+                    command.getTable())) {
+                throw tableNotFound();
+            }
+        }
+
+        // No need to check at resolve time since there is no actual state involved. If Deephaven exposes the notion
+        // of keys, this will need to behave more like QueryBase where there is a handle-based ticket and some sort
+        // state maintained. It is also incorrect to perform the same checkForFlightInfo at resolve time because the
+        // state of the server may have changed between getInfo and doGet/doExchange, and getInfo should still be valid
+        // for client.
+        // @Override
+        // void checkForResolve(CommandGetExportedKeys command) {
```

Comment:
Same about the commented method.

I'd also be cool with the default impl having such an in-line comment that most impls won't override and this is why.

Reply:
I've centralized the comments.